<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="node_name" default="ctlaser"/>
	<arg name="laser_ip" default="146.164.53.228"/>
	<arg name="laser_port" default="2101"/>
	<arg name="target_lights" default="false"/>
	<arg name="frequency" default="10"/>
	<arg name="required" default="true"/>
	<arg name="focal_distance" default="0.45"/>

    <node name="$(arg node_name)"
	    pkg="ctlaser_driver"
		type="ctlaser_driver_node"
		output="screen"
		required="$(arg required)"
		args="_laser_ip:=$(arg laser_ip)
		      _laser_port:=$(arg laser_port)
			  _target_lights:=$(arg target_lights)
			  _frequency:=$(arg frequency)">
	</node>

    <node name="$(arg node_name)_focal_point_broadcaster"
	    pkg="tf"
		type="static_transform_publisher"
		args="0 0 $(arg focal_distance) 0 0 0 \$(arg node_name)_frame \$(arg node_name)_focal_frame 100">
	</node>
</launch>

